manual_hook.rb

```diff
@@ -20,7 +18,14 @@ def setup_dns(domain, txt_challenge)
        puts "Found #{resp.strings[0]}. no match."
      end
     }
+
     if !resolved
+
+     puts "Create TXT record for the domain: \"#{acme_domain}\". TXT record:"
```

Comment:
because this is inside the `until` loop this will be displayed on each iteration (which depending on how fast DNS propagates, may be multiple times. This is not ideal.